docs/xls-56d-batch-transactions/reference/common-fields.md

```diff
@@ -17,7 +17,6 @@ Every transaction has the same set of common fields, plus additional fields base
 | `Fee`                | String           | Amount            | _(Required; [auto-fillable](https://xrpl.org/docs/references/protocol/transactions/common-fields#auto-fillable-fields))_ Integer amount of XRP, in drops, to be destroyed as a cost for distributing this transaction to the network. Some transaction types have different minimum requirements. See [Transaction Cost](https://xrpl.org/docs/concepts/transactions/transaction-cost/) for details. |
 | `Sequence`           | Number           | UInt32            | _(Required; [auto-fillable](https://xrpl.org/docs/references/protocol/transactions/common-fields#auto-fillable-fields))_ The [sequence number](https://xrpl.org/docs/references/protocol/data-types/basic-data-types/#account-sequence) of the account sending the transaction. A transaction is only valid if the `Sequence` number is exactly 1 greater than the previous transaction from the same account. The special case `0` means the transaction is using a [Ticket](https://xrpl.org/docs/concepts/accounts/tickets/) instead _(Added by the [TicketBatch amendment](https://xrpl.org/resources/known-amendments/#ticketbatch).)_. |
 | [`AccountTxnID`](#accounttxnid) | String | Hash256          | _(Optional)_ Hash value identifying another transaction. If provided, this transaction is only valid if the sending account's previously sent transaction matches the provided hash. |
-| [`BatchTxn`](#batchtxn)         | object | STObject         | _(Optional)_ The `BatchTxn` inner object must be included in any inner transaction of a `Batch` transaction. |
 | [`Flags`](#flags-field) | Number        | UInt32            | _(Optional)_ Set of bit-flags for this transaction. |
 | `LastLedgerSequence` | Number           | UInt32            | _(Optional; strongly recommended)_ Highest ledger index this transaction can appear in. Specifying this field places a strict upper limit on how long the transaction can wait to be validated or rejected. See [Reliable Transaction Submission](https://xrpl.org/docs/concepts/transactions/reliable-transaction-submission/) for more details. |
```

Comment:
For consistency, suggest noting "_(Optional)_" and then including a note about strongly recommended. 
Also, optional and strongly recommended sends a mixed message. Adding additional information/context on when this field is strongly recommended may be helpful.

Reply:
As that's a problem in the main docs, I suggest that we punt here and create a Git issue on xrpl.org.